components/molecules/CollaborationSummaryCard/collaboration-summary-card.tsx

```diff
@@ -0,0 +1,50 @@
+import { TbMessageCode } from "react-icons/tb";
+
+import Avatar from "components/atoms/Avatar/avatar";
+import React from "react";
+import { CollaborationRequestObject } from "../CollaborationCard/collaboration-card";
+import { BiMessage } from "react-icons/bi";
+
+interface CollaborationSummaryCardProps {
+  requests: CollaborationRequestObject[];
+  connectionsCount?: number;
+  messagesCount?: number;
+}
+const CollaborationSummaryCard = ({ requests, connectionsCount, messagesCount }: CollaborationSummaryCardProps) => {
+  return (
+    <div className="bg-white flex flex-col gap-4 rounded-2xl border-light-slate-6  p-4 border max-w-[41.625rem]">
```

Comment:
Could you please not use max-w-[41.625], but use max-w-2xl from tailwind is 42 rem
```suggestion
    <div className="bg-white flex flex-col gap-4 rounded-2xl border-light-slate-6  p-4 border max-w-2xl">
```